sidebars/docs.js

```diff
@@ -11,18 +11,85 @@
 
 module.exports = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
-  tutorialSidebar: [{
-    type: 'autogenerated', dirName: '.',
-  }],
+  // tutorialSidebar: [{
+  //   type: 'autogenerated', dirName: '.',
+  // }],
 
   // But you can create a sidebar manually
-  /*
+
   tutorialSidebar: [
+    'introduction',
+    'roadmap',
+    {
+      type: 'category',
+      label: 'Design Document',
+      collapsible: true,
+      collapsed: false,
+      items: [
+        {
+          type: 'category',
+          label: 'Event Handling and Integration',
+          collapsible: true,
+          collapsed: false,
+          items: [
+            {
+              type: 'autogenerated',
+              dirName: 'design-document/01-event-handling-and-integration',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'Observability',
+          collapsible: true,
+          collapsed: false,
+          items: [
+            {
+              type: 'autogenerated',
+              dirName: 'design-document/02-observability',
+            },
+          ],
+        },
+        'design-document/schema-registry',
+        'design-document/spi',
+        'design-document/stream',
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Installation and Deployment',
+      collapsible: true,
```

Comment:
The "Installation and Deployment" category should be in front of "Design Document". It is not recommended to change the original order of columns. 

![image](https://github.com/apache/eventmesh-site/assets/41445332/766a8f3b-8e2d-45a7-b064-de5b4ce0bf47)

Reply:
Fixed in https://github.com/apache/eventmesh-site/pull/152.